trino/Dockerfile

```diff
@@ -78,5 +84,53 @@ COPY shared/log4shell_scanner /bin/log4shell_scanner
 RUN /bin/log4shell_scanner s /stackable/trino-server-${PRODUCT}
 # ===
 
+FROM stackable/image/java-devel AS jmx-exporter-builder
```

Comment:
```suggestion
# Not tagging base image because it is built as part of the same process
# hadolint ignore=DL3006
FROM stackable/image/java-devel AS jmx-exporter-builder
```